packages/data/platform/prophecies_repository_flutter/lib/data/prophecies.json

```diff
@@ -15,17 +15,17 @@
   },
   {
     "model": {
-          "id": "2",
-          "name": "Внутренняя энергия",
-          "value": 4.9
+      "id": "2",
+      "name": "Внутренняя сила",
```

Comment:
Did we discuss how we are going to translate those?

Reply:
This is hardocoded local file. This values will be getted from algorithm and displayed with localization. But for now, it is ordinary local json file.